processing/src/main/java/org/apache/druid/query/aggregation/post/ExpressionPostAggregator.java

```diff
@@ -190,7 +218,7 @@ public String toString()
   {
     /**
      * Ensures the following order: numeric > NaN > Infinite.
-     *
+     * <p>
```

Comment:
Could you please remove this html tag? If intellij did this, there is a setting that controls it:

```
Code Style -> Java -> JavaDoc tab -> Generate "<p>" on empty lines
```

Reply:
Fixed! Thanks for the feedback.